doc/conf.py

```diff
@@ -36,6 +36,7 @@
     "sphinx_copybutton",
     "myst_parser",
     "sphinx_design",
+    "exasol.toolbox.sphinx.multiversion",
```

Comment:
this doesn't look like an update of python-toolbox

Reply:
No it isn't, this will disappear. I plan to merge https://github.com/exasol/error-reporting-python/pull/53 first ;)

Reply:
Also it looks like I forgot to push all changes :sweat_smile: